Packages/ca.tekly.logger/Runtime/LogDestinations/UnityLogDestination.cs

```diff
@@ -82,6 +89,38 @@ public void LogMessage(TkLogMessage message, Object context, LogSource logSource
             }
         }
 
+#if UNITY_EDITOR
+        private string LinkStacktraceToCode(string stackTrace)
+        {
+            if (string.IsNullOrEmpty(stackTrace)) {
+                return stackTrace;
+            }
+
+            var sb = new StringBuilder(stackTrace.Length * 2);
+            var rows = stackTrace.Split('\n');
+
+            foreach (var row in rows) {
+                var match = Regex.Match(row, "(.* \\(at )(([^\\/].*):([0-9]+))", RegexOptions.RightToLeft);
```

Comment:
I think this can be faster by precompiling the regex

Reply:
Done.